actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -571,6 +571,54 @@ def current_page?(options, check_parameters: false)
         end
       end
 
+      # Creates an SMS anchor link tag to the specified +phone_number+, which is
+      # also used as the name of the link unless +name+ is specified. Additional
+      # HTML attributes for the link can be passed in +html_options+.
+      #
+      # When clicked, an SMS message is prepopulated with the passed phone number
+      #  and optional +body+ value.
```

Comment:
![image](https://user-images.githubusercontent.com/2091837/61754702-dfdd9680-ad79-11e9-857c-c3aa0a4ad6fb.png)

Watch out for indentation here (being parsed as a code block)

Reply:
Good catch. Fixed by 0e149abbd5affc43f80a5c9bf8b7a7c2a5905d53.